src/ui/public/index_patterns/_index_pattern.js

```diff
@@ -95,12 +95,10 @@ export function IndexPatternProvider(Private, config, Promise, confirmModalPromi
 
   function updateFromElasticSearch(indexPattern, response, forceFieldRefresh = false) {
     if (!response.found) {
-      const markdownSaveId = indexPattern.id.replace('*', '%2A');
-
       throw new SavedObjectNotFound(
         type,
         indexPattern.id,
-        kbnUrl.eval('#/management/kibana/index?id={{id}}&name=', { id: markdownSaveId })
+        '#/management/kibana/index',
```

Comment:
I made this change because the `id` query string parameter is used to populate the index pattern text input. However, this doesn't make sense because the ID is something like `1ac33700-9417-11e8-b8ea-8ff973531737` and the input is for the index pattern name, e.g. `logstash-*`.